src/supervisor.tex

```diff
@@ -2108,3 +2117,144 @@ \subsection{Addressing and Memory Protection}
 The algorithm for virtual-to-physical address translation is the same
 as in Section~\ref{sv32algorithm}, except LEVELS equals 5 and PTESIZE
 equals 8.
+
+\chapter{``Svnapot'' Standard Extension for NAPOT Translation Contiguity, Version 0.1}
+\label{svnapot}
+
+In Sv39, Sv48, and Sv57, when Svnapot is enabled and a PTE has N=1, the PTE represents a
```

Comment:
> when Svnapot is enabled

Does this temporarily need to be stated as "via a TBD mechanism"?  Or is this meant to say "is implemented"?